### ISSUE TITLE
Settings not visible/readable in dark mode

### ISSUE THREAD

**Issue by yokuha** [dennisverspuij/tb-correctidentity#38]
This is TB 115 on macOS – cannot configure as I cannot read settings and info.
![Screenshot 2023-10-17 at 09 47 37](https://github.com/dennisverspuij/tb-correctidentity/assets/124187637/d609c17a-04aa-4c3e-b58f-9813aa52dce1)

**Comment by alkuzad** [comment]
This is true for every addon, not only this one. It's a problem of Thunderbird itself
